ts/UpdateAppModal.tsx

```diff
@@ -0,0 +1,138 @@
+/**
+ * A screen to invite the user to update the app because current version is not supported yet
+ *
+ */
+
+import { Button, Container, H2, Text, View } from "native-base";
+import * as React from "react";
+import {
+  BackHandler,
+  Image,
+  Linking,
+  Modal,
+  Platform,
+  StyleSheet
+} from "react-native";
+import { connect } from "react-redux";
+import BaseScreenComponent from "./components/screens/BaseScreenComponent";
+import FooterWithButtons from "./components/ui/FooterWithButtons";
+import I18n from "./i18n";
+import customVariables from "./theme/variables";
+import { showToast } from "./utils/showToast";
+
+const styles = StyleSheet.create({
+  text: {
+    marginTop: customVariables.contentPadding,
+    fontSize: 18
+  },
+  container: {
+    margin: customVariables.contentPadding,
+    flex: 1,
+    alignItems: "flex-start"
+  },
+  img: {
+    marginTop: customVariables.contentPaddingLarge,
+    alignSelf: "center"
+  }
+});
+
+const openAppStore = () => {
+  const storeUrl = Platform.select({
+    ios: "itms-apps://itunes.apple.com/it/app/testflight/id899247664?mt=8",
+    android: "market://details?id=it.teamdigitale.app.italiaapp"
+  });
+  Linking.openURL(storeUrl).catch(() => {
+    // Display an toast with an error message
+    showToast(I18n.t("msgErrorUpdateApp"), "danger");
```

Comment:
It won't be displayed because the toast is rendered under this overlay screen. (try to your self by running on the ios emulator)
You could fix it by providing an error text inside this screen.
It should be shown and removed automatically (wait 5 seconds to hide it)

Reply:
you're right!

Reply:
This is the result when an error occurred on click the button 'Update app', the red label hides after 5 seconds

 <table style="width:100%">
  <tr>
    <th>iOS</th>
    <th>Android</th>
  </tr>
  <tr>
    <td width="300"><img src="https://user-images.githubusercontent.com/7993551/72333369-fb3bb580-36bb-11ea-999a-60886826147f.png"/></td>
    <td width="300"><img src="https://user-images.githubusercontent.com/7993551/72333448-17d7ed80-36bc-11ea-87c9-0891f52759a8.jpg"/></td>
  </tr>
</table>

Reply:
maybe this trick could work https://github.com/GeekyAnts/NativeBase/issues/985#issuecomment-398688935